proposals/low-level-struct-improvements.md

```diff
@@ -245,8 +249,6 @@ Span<int> ScopedLocalExamples()
 
 Other uses for `scoped` on locals are discussed [below](#examples-scoped-locals).
 
-When `scoped` is applied to an instance method the `this` parameter will have the type `scoped ref T`. By default this is redundant as `scoped ref T` is the default type of `this`. It is useful in the case the `struct` is declared as `unscoped` (detailed [below](#return-fields-by-ref)).
```

Comment:
As written felt like I was just mostly restating previous stuff. Expanded section above and removed this.